### ISSUE TITLE
Operational issues

### ISSUE THREAD

**Issue by LYD15836253141** [aim-uofa/PM-Loss#8]
Hello! Thank you very much for your excellent work! I have successfully run the test results of DepthSplat and configured the PM-Loss environment based on the readme you provided. However, I don't understand how to run the PM-Loss command to obtain the test results, so as to facilitate comparison with the test results of DepthSplat and highlight the improvement effect. Could you offer some help? Another question is: Can this solution be applied to the SPFSplat solution? Looking forward to your reply!

**Comment by LYD15836253141** [comment]
> Hi, [@LYD15836253141](https://github.com/LYD15836253141) ! Thanks for your interest in our work.
> 
> To evaluate the performance of PM-Loss, you can follow the instructions in our README. Please download the file from our Hugging Face repository here: https://huggingface.co/dc-walker/PM-Loss. After that, you can load our pre-trained weights by adding to your command.`pmloss.ckpt``checkpointing.pretrained_model=checkpoints/pmloss.ckpt`
> 
> If you'd like to reproduce our training process, you can refer to the training section in our README: https://github.com/aim-uofa/PM-Loss?tab=readme-ov-file#Training.
> 
> As for your question about applying our method to SPFSplat, you may want to check if SPFSplat suffers from the "fragmented floaters" issue. If it does, we believe that PM-Loss could be beneficial.
> 
> Hope this helps!

Thank you very much for your reply! As you said, I have run the pp-loss evaluation command, but I cannot obtain rendered videos, depth maps, and.ply files like DepthSplat. I can only get some files containing evaluation metrics in the outputs folder, even though I have set up saving the above content in main.py. Could you provide the command for exporting rendered videos, depth maps and.ply files using PM-Loss? It is similar to the Rendering Video command in the Readme file of DepthSplat. Looking forward to your reply！
